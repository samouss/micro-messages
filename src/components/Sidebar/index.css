.Sidebar {
  display: flex;
  height: 100%;
  width: 400px;
  border-right: 1px solid #E5E5E5;
  overflow-y: scroll;
}
