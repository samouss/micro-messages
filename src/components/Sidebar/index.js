// @flow

import React from 'react';
import './index.css';

type Props = {
  children: React.Element<*>,
};

const Sidebar = ({ children }: Props) => (
  <div styleName="container">
    {children}
  </div>
);

export default Sidebar;
