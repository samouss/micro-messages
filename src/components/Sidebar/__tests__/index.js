// @flow

import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from '../index';

describe('<Sidebar />', () => {
  it('expect to render the component correctly', () => {
    const component = shallow(
      <Sidebar>
        <div>Some content</div>
      </Sidebar>,
    );

    expect(component).toMatchSnapshot();
  });
});
