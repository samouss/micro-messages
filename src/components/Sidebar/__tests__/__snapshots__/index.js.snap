// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar /> expect to render the component correctly 1`] = `
<div
  styleName="container"
>
  <div>
    Some content
  </div>
</div>
`;
